src/main/java/at/hannibal2/skyhanni/features/inventory/AuctionsHighlighter.kt

```diff
@@ -63,4 +64,10 @@ class AuctionsHighlighter {
             }
         }
     }
+
+    @SubscribeEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(24, "inventory.highlightAuctions", "inventory.auctions.highlightAuctions")
```

Comment:
You moved three existing config options, you only fixed two in here.
also you need to increase the version number by one

Reply:
The other one is in AuctionHouseCopyUnderbidPrice.kt